### ISSUE TITLE
Feature: Hashtag autocomplete

### ISSUE THREAD

**Issue by lionleaf** [lionleaf/dwitter#335]
Having an autocomplete of hashtags that tells you how many dweets are in a given hashtag would be helpful to make sure people pick the same version of a hashtag. #trees vs #tree, #multiDweet vs #multi_dweet etc.

This would require some front end js to recognize when tweets are being typed along with an api that returns number of dweets for a hashtag (and similar hashtags?)

**Comment by kipkat** [comment]
Yeah, in my opinion, this would require some front end js and using a special api for hashtags is the best solution.
